src/App.tsx

```diff
@@ -33,6 +61,13 @@ export default function App() {
           <Route path="/google" element={<Google />} />
         </Routes>
       </Main>
+      <CookieConsent
+        enableDeclineButton
+        onAccept={handleAcceptCookie}
+        onDecline={handleDeclineCookie}
+      >
+        This website uses cookies to enhance the user experience.
```

Comment:
Ici le message est très important parce que sinon on va passer pour des cons si c'est mal communiqué.

J'aurais mis qqch dans cet esprit là : 

"We use a Google Analytics tag to estimate the volume of users that visited Avastar. We only store an anonymised IP address as well a device-id, whenever a new visitor come to our app."